subsys/net/lib/lwm2m/Kconfig.ipso.current_sensor

```diff
@@ -0,0 +1,37 @@
+# Copyright (c) 2021 Laird Connectivity
```

Comment:
Can we just keep it together with other IPSO configurations? I'd rather avoid multiplying Kconfig files, and it'd be consistent with current approach.